src/index.html

```diff
@@ -11,12 +11,101 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo moyo header"
+        class="logo"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#1"
+              class="nav__link nav__link--active is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#2"
+              class="nav__link"
+            >
+              SAMSUNG
```

Comment:
use css everywhere for uppercase
```suggestion
              samsung
```